Gruntfile.js

```diff
@@ -36,7 +36,9 @@ module.exports = function(grunt) {
             "vendor/underscore-1.8.3.js",
             "vendor/backbone-1.1.2.js",
             "vendor/jquery.dataTables-1.9.4.js",
-            "vendor/mock-ajax.js"
+            "vendor/mock-ajax.js",
+            "vendor/bootstrap-3.1.1-dist/js/bootstrap.js",
```

Comment:
If only needed for the specs, we could stub the smallest amount needed (vendor/mock-bootstrap.js) ?  But if we want examples to show the actual filtering working, which would be nice.. I don't have any suggestion for this pass